src/benchq/resource_estimation/of_modified.py

```diff
@@ -0,0 +1,222 @@
+#   Copyright 2017 The OpenFermion Developers
+#   Modifications copyright 2023 Zapata Computing, Inc. for compatibility reasons.
```

Comment:
Let's change `for compatibility reasons` to something like `for purposes of giving users more control over physical costing parameters`.

Reply:
Sounds better!